### PR TITLE
[DMN] *: use cluster name instead of cluster id

### PR DESCRIPTION
**Pull request by tiancaiamao**

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -118,13 +118,13 @@ message AskSplitResponse {
...
     optional bytes uuid                = 1;
-    optional uint64 cluster_id         = 2;
+    optional string cluster_name         = 2;
 }
 
 message ResponseHeader {
     // 16 bytes, to distinguish request.  
     optional bytes uuid                = 1;
-    optional uint64 cluster_id         = 2;
+    optional string cluster_name         = 2;
```
Comment (siddontang) [inline proto/pdpb.proto]: alignment

File: proto/pdpb.proto [hunk 2]
```diff
@@ -118,13 +118,13 @@ message AskSplitResponse {
 message RequestHeader {
     // 16 bytes, to distinguish request.  
     optional bytes uuid                = 1;
-    optional uint64 cluster_id         = 2;
+    optional string cluster_name         = 2;
```
Comment (siddontang) [inline proto/pdpb.proto]: alignment

File: proto/pdpb.proto [hunk 3]
```diff
@@ -68,10 +68,10 @@ message AllocIdResponse {
...
 
 message GetMetaRequest {
-    optional MetaType meta_type    = 1;
-    optional uint64 store_id       = 2;
-    optional bytes region_key      = 3;
-    optional uint64 cluster_id     = 4;
+	optional MetaType meta_type    = 1;
+	optional uint64 store_id       = 2;
+	optional bytes region_key      = 3;
+	optional string cluster_name   = 4;
```
Comment (siddontang) [inline proto/pdpb.proto]: I think we should remove this and use another request like GetClusterConfig

File: proto/pdpb.proto (hunk 1 above)
Comment (siddontang) [inline proto/pdpb.proto]: we can remove this field now.

File: proto/pdpb.proto (hunk 3 above)
Comment (siddontang) [inline proto/pdpb.proto]: @qiuyesuifeng

File: src/coprocessor.rs [hunk 4]
```diff
@@ -175,11 +171,11 @@ impl ::protobuf::Message for KeyRange {
         self.cached_size.get()
     }
 
-    fn get_unknown_fields(&self) -> &::protobuf::UnknownFields {
+    fn get_unknown_fields<'s>(&'s self) -> &'s ::protobuf::UnknownFields {
```
Comment (ngaut) [inline src/coprocessor.rs]: Please update your protubuf

File: proto/pdpb.proto (hunk 3 above)
Comment (qiuyesuifeng) [inline proto/pdpb.proto]: Seems now we can split meta into cluster config, store and region three different requests, that will be clearer.

Comment (siddontang) [inline proto/pdpb.proto]: +1

```
get store
get region
get cluster config
put store
put cluster config
```

then we can remove MetaType too.

@ngaut

**Pull request closed without merging**